audit/forgerock-audit-core/src/main/java/org/forgerock/audit/events/AuditEventBuilder.java

```diff
@@ -160,8 +160,22 @@ public final T eventName(String name) {
      * @return this builder
      */
     public final T timestamp(long timestamp) {
+        return timestamp(timestamp, true);
```

Comment:
It is better to keep the default value for compatibility at least at the API level.
As a result, there is no need to add arguments in unit tests.

Reply:
In short, the default is to get timestamps in UTC?

Reply:
yup.

Reply:
Woops... OK.